### DISCUSSION TITLE
delucidazione su AIUTANTI

### DISCUSSION THREAD

**Discussion by effe-7** [indomus/forum#2708] [HUB: Home Assistant]
Buongiorno, 
sicuramente mi sta sfuggendo qualcosa,
perché avrei bisogno di capire meglio le funzionalità degli "aiutanti" 
nello specifico elementi come  il "pulsante" ed il "commutatore" che io ho interpretato come la possibilità di avere dispositivi virtuali ma poi non riesco a recuperarli nei dispostivi per fare le automazioni.

grazie

**Comment by indomus** [comment]
> Buongiorno, sicuramente mi sta sfuggendo qualcosa, perché avrei bisogno di capire meglio le funzionalità degli "aiutanti" nello specifico elementi come il "pulsante" ed il "commutatore" che io ho interpretato come la possibilità di avere dispositivi virtuali ma poi non riesco a recuperarli nei dispostivi per fare le automazioni.

sono semplicissime entità che poi ti devi giostrare come meglio credi. un pulsante non è altro che un [button](https://www.home-assistant.io/integrations/button/), e così via.

**Comment by effe-7** [comment]
ok, e come posso richiamarli per avviare un'automazione?

**Reply by indomus** [reply]
devi guardare i servizi che sono indicati nella pagina che ti ho linkato (almeno per i button; per altri devi guardare le relative schede di documentazione).
